exporters/trace/stackdriver/README.md

```diff
@@ -103,6 +111,12 @@ In all the versions before '0.9.1' the Stackdriver Trace exporter was implemente
 API][stackdriver-v1-api-url] which is not fully compatible with the OpenCensus data model. Trace 
 events like Annotations and NetworkEvents will be dropped.
 
+### Why do I get a StatusRuntimeException: NOT_FOUND: Requested entity was not found?
```

Comment:
It would help to add quotes around the error message.

Reply:
Done.